python-sdk/src/astro/sql/operators/append.py

```diff
@@ -63,6 +63,9 @@ def __init__(
 
     def execute(self, context: Context) -> BaseTable:  # skipcq: PYL-W0613
         db = create_database(self.target_table.conn_id, table=self.source_table)
```

Comment:
Can we move this line below the if condition, so that we avoid another DB call with that method in case tables do not belong to the same datasource?